docs/buildah.md

```diff
@@ -89,5 +92,7 @@ Print the version
 | buildah-run(1)        | Run a command inside of the container.                                                               |
 | buildah-tag(1)        | Add an additional name to a local image.                                                             |
 | buildah-umount(1)     | Unmount a working container's root file system.                                                      |
-| buildah-version(1)    | Display the Buildah Version Information
-                                               |
+| buildah-version(1)    | Display the Buildah Version Information                                    |
+| storage.conf(5) | Syntax of Container Storage configuration file
+ |
+| |                                                  |
```

Comment:
Something funky going on here , there's a bunch of random lines at the bottom of the page if you hit the "View" button.  I think line 98 needs to go away.

Reply:
@TomSweeneyRedHat I am trying to figure out how we cat get this to look good in both markdown and man page.  Adding these lines caused the bottom of the man page to look better.

Try compiling the docs as a man page and then run man on it. WIth this patch we get an empty thable line, without this patch we get a table line contianer   .TH

Reply:
I figured this is the least ugly way to clean that up.  If you want to play with the markdown to see if you can get something cleaner, I am all for it.